pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "aladdin"
-version = "1.25.15.0"
+version = "1.25.16.0"
```

Comment:
we'll want to leave this as 1.25.15 (should match k8s version), and update only the last digit